dv/formal/conductor.py

```diff
@@ -0,0 +1,802 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# Original author: Louis-Emile Ploix
+# SPDX-License-Identifier: Apache-2.0
+
+# This program handles the running of proof processes. It can discover efficient proofs, schedule them, find
+# counterexamples, and analyse the results.
+
+import asyncio
+import argparse
+import re
+import subprocess
+import time
+import json
+import os
+from math import floor
+from datetime import datetime
+import hashlib
+
+# --------------------- Utilities ----------------------
+
+PROOFLOG = os.environ["PROOFLOG"] if "PROOFLOG" in os.environ else "prooflog.txt"
+LOGFILE = os.environ["LOGFILE"] if "LOGFILE" in os.environ else "logfile.txt"
```

Comment:
Maybe make these environment names a bit more unique to avoid conflicts. Like IBEX_PROOF_LOG and IBEX_PROOF_LOG_FILE

Reply:
You could also simplify this using `.get` with a default value i.e.

```python
os.environ.get("PROOFLOG", "prooflog.txt")
os.environ.get("LOGFILE", "logfile.txt")
```